CSharp/Project/MVVMtpl/MVVMtpl.Shared/ViewModels/ExampleViewModel.cs

```diff
@@ -25,7 +25,7 @@ public ExampleViewModel(NavigationService navigationService, MessagingService me
             this.navigationService = navigationService;
             this.messagingService = messagingService;
 
-            messagingService.Register();
+            //messagingService.Register();
```

Comment:
I'd be reluctant to have this commented-out (this is the main reason we have a git, so we don't have this kind of things). If it's trully not necessary, just delete it, if it was a test, please reset it back to normal. Nevertheless, please check with @CKGrafico to decide if it's unnecessary, or why would you delete it.